app/src/main/java/com/baijum/ukufretboard/ui/ScalePracticeEarTraining.kt

```
@@ -0,0 +1,192 @@
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.baijum.ukufretboard.R
+import com.baijum.ukufretboard.data.LearningStats
+import com.baijum.ukufretboard.data.Notes
+import com.baijum.ukufretboard.viewmodel.LearningProgressViewModel
+import com.baijum.ukufretboard.viewmodel.ScalePracticeUiState
+import com.baijum.ukufretboard.viewmodel.ScalePracticeViewModel
+
+@Composable
+internal fun EarTrainingContent(
```

detekt (1.23.8)

[warning] The function EarTrainingContent appears to be too complex based on Cognitive Complexity (complexity: 42). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)